tests/test_driver.py

```diff
@@ -98,6 +98,7 @@ def test_prepare(self, driver, alice_pubkey):
         assert not transaction['metadata']
 
     def test_fulfill(self, driver, alice_keypair, unsigned_transaction):
+        from sha3 import sha3_256
```

Comment:
we can import this at the top, as sha3 is an external lib, and is not under test

as an aside sha3 is now in the standard lib for Python 3.6+: https://docs.python.org/3/library/hashlib.html#hash-algorithms